src/test/resources/testdata/rest/customobjectinstances/personcustomobjectinstance30-data.txt

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
There's already a data file in the master branch for this object that we missed because our branch is behind